bool cart
geometry_msgs/Pose pose
float64[] joints

bool moveHand
float64 angle